packages/sit-onyx/src/types/utils.ts

```diff
@@ -2,3 +2,6 @@
  * Recursive / deep implementation of TypeScript's built-in `Partial<T>` type.
  */
 export type DeepPartial<T> = T extends object ? { [P in keyof T]?: DeepPartial<T[P]> } : T;
+
+export type Equals<A extends never> = A;
```

Comment:
do you also want to leave the comment to remove it wiht the workaround or can this also be useful for other use cases? :)